packages/devextreme/js/data/abstract_store.d.ts

```diff
@@ -1,219 +1,31 @@
-// eslint-disable-next-line max-classes-per-file
 import {
-  DxPromise,
   DxExtendedPromise,
 } from '../core/utils/deferred';
-import { DeepPartial } from '../core/index';
+import { Store, Options as StoreOptions } from './store';
 import {
-  FilterDescriptor,
-  GroupDescriptor,
   LoadOptions,
 } from './index';
 
 export type Options<
     TItem = any,
     TKey = any,
-> = StoreOptions<TItem, TKey>;
+> = AbstractStoreOptions<TItem, TKey>;
 
 /**
  * @namespace DevExpress.data
  * @deprecated Use Options instead
- * @docid
- * @hidden
  */
-export interface StoreOptions<
+export interface AbstractStoreOptions<
```

Comment:
We need an alias for `StoreOptions` for back-compat.
And we have no reason to add deprecated alias for the new `AbstractStoreOptions` since no one can have existing imports of it.

```suggestion
export type Options<
```

Reply:
Fixed (partially). Removed most of doc tags and "export" from AbstractStoreOptions, but kept it because of failure on ts discovering on a type with an anonymous member. I also suggest not to add an alias, because moved StoreOptions is more private than AbstractStoreOptions even.